bad_code.java

```diff
@@ -0,0 +1,54 @@
+public class BadCodeExample {
```

Comment:
Use `@SuppressWarnings` annotation to suppress unneeded warnings, if any. [medium]